ex2/ex_2.cs

```diff
@@ -19,68 +19,63 @@ namespace Ex2
      * 7. Add to BankAccount the operator + so we could use it to directly update the amount of money in the account.
      * 8. Write a method called IsPalindrome which receives an integer and checks whether it is a palindrome.
      */
-    class Answers
+    class Program
     {
         // 1
-        public enum AccountType { Checking, Deposit }
+        enum AccountType { Checking, Deposit }
 
         // 2
-        public struct BankAccount : IComparable<BankAccount>
+        struct BankAccount : IComparable<BankAccount>
         { 
-            public string owner; public int moneyInAccount; public AccountType accType; 
+            internal string owner; 
```

Comment:
Why `internal` and not `public`? Do you know the difference?

Reply:
I know the difference. The question of accessibility to these fields is not relevant to the questions in the exercise, both access modifiers will give the same results to the relevant questions. 
To solve the questions, the lowest level of accessibility necessary is `internal`, but you probably see the role of `BankAccount` struct different than how I do. If you think these should be `public`, I will change them.